targets/simple_switch_grpc/switch_runner.cpp

```diff
@@ -153,17 +175,10 @@ class DataplaneInterfaceServiceImpl
   }
 
   void transmit_fn_(port_t port_num, const char *buffer, int len) override {
```

Comment:
please call `my_transmit_fn` to make it a valid `DevMgrIface` implementation

Reply:
Done